cluster-autoscaler/vendor/modules.txt

```diff
@@ -229,7 +229,7 @@ github.com/euank/go-kmsg-parser/kmsgparser
 github.com/evanphx/json-patch
 # github.com/fsnotify/fsnotify v1.4.9 => github.com/fsnotify/fsnotify v1.4.9
 github.com/fsnotify/fsnotify
-# github.com/gardener/machine-controller-manager v0.33.0 => /Users/i331134/go/src/github.com/gardener/machine-controller-manager
+# github.com/gardener/machine-controller-manager v0.35.0 => /Users/i341888/Go/src/github.com/gardener/machine-controller-manager
```

Comment:
Can you check if this also works with the personal fork link in `mod-extra`, to avoid having a number here?
- I missed out on this earlier in my changes as it seems, sorry about that.

Reply:
I tried making this change even now and even earlier. However, I think the dependency manager expects release version tags. 

```
go.mod-extra:7:3: require github.com/gardener/machine-controller-manager: version "e00847dff26e9044d0f18270ac3e0ff78bd872fb" invalid: must be of the form v1.2.3
```

 So I will skip this for now. However, let's fix this permanently by vendoring the right version of k8s (v1.18) on MCM in the future so that we don't run into this issue.